src/midi.rs

```diff
@@ -181,7 +181,7 @@ pub struct Program(u8);
 
 impl From<u8> for Program {
     fn from(value: u8) -> Self {
-        assert!(value <= 127);
+        debug_assert!(value <= 127);
         Program(value)
```

Comment:
should this at least have `.min(127)` there?

Reply:
Good point. I'll fix that

Reply:
I have a new PR that adds `const fn new` to all these types and clamps them appropriately.. maybe it would make sense to add `debug_assert!` to the various types AND call `new(value)` so we get the clamping and also get debug assertions?

Maybe I'll just add that to my PR?